### DISCUSSION TITLE
One more question: Unrecognized DVD becomes ISO?

### DISCUSSION THREAD

**Discussion by kubycsolutions** [automatic-ripping-machine/automatic-ripping-machine#1143] [Q&A]
I've got a Samsung DVD recorder which permits dubbing of (non-protected) VHS tapes to recordable DVD. These DVDs seem to play successfully with the PC software I've tried them with, even in the device's default mode which is known to be incompatible with some players. (I think it leaves the disk un-Finalized so more tracks/sessions can be added, or some similar detail.)

When I hand one of these to ARM, it seems to be treating it as a data disk; I wind up with the fle `./dvds/completed/unidentified/SAMSUNG_DVD_RECORDER_VOLUME/SAMSUNG_DVD_RECORDER_VOLUME.iso`.

Logfiles  attached.  That confirms that this is indeed being treated as a data disk, which makes .iso a perfectly reasonable choice. But it'd be nice if we could persuade ARM to handle these as DVD rips.

[SAMSUNG_DVD_RECORDER_VOLUME_171779164488.log](https://github.com/user-attachments/files/15744834/SAMSUNG_DVD_RECORDER_VOLUME_171779164488.log)
[SAMSUNG_DVD_RECORDER_VOLUME.log](https://github.com/user-attachments/files/15744835/SAMSUNG_DVD_RECORDER_VOLUME.log)

Obviously, let me know if there's anything I can run against this DVDR to help figure out whether ARM can be configured to be happier with it.

**Secondary isue: ** Note that the filename is apparently taken from the disk label. Which makes sense... except that all the disks recorded by my VHS dubber will have that same label, as far as I know. Could/should the DVD hash and/or a timestamp be included as part of the filename when putting something in  `/unidentified/`, so these wouldn't risk overwriting each other? User would still have to figure out which is which, but...

**Comment by kubycsolutions** [comment]
(Note: I am very aware this is nitpick and edge case on open-source code. I's a relatively simple code change I'm willing to try it and test it. If it's complicated, I won't pester you about it. Just a WIBNI suggestion, which might already have a simple solution...)

**Reply by microtechno9000** [reply]
This is something we are working on in the  background as currently ARM isn't super flexible. See #815
